tutorial06/my-ed-config.yaml

```diff
@@ -8,8 +8,8 @@ plugins:
   - name: localization
     type: ed_localization/localization
     parameters:
-        robot_name: amigo
-        initial_pose_topic: /amigo/initialpose
+        robot_name: hero
```

Comment:
In the future we should look at using an open source robot model for the tutorials. But for now it is okay if we use hero internally.

Reply:
Oh, we actually decided to not make it hero specific. So slipped through